src/modules/openPopup.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-use-before-define */
+
+import { likeCountes } from './getCounts.js';
+
+export function openPopup(id) {
+  function fetchPokemonsPopup(id) {
+    fetch(`https://pokeapi.co/api/v2/pokemon/${id}`)
+      .then((response) => response.json())
+      .then((pokemonInfo) => {
+        renderPkmnPopup(pokemonInfo);
+      });
+  }
+
+  function createTypes(types, ul) {
+    types.forEach((type) => {
+      const typeLi = document.createElement('li');
+      typeLi.innerText = type.type.name;
+      ul.append(typeLi);
+    });
+  }
```

Comment:
Please reuse the functions as there is already a function created to render poke types